tool/jt.rb

```diff
@@ -1152,6 +1154,17 @@ def test_tck(*args)
     end
   end
   private :test_tck
+  
+  def gem_test_pack
+    test_pack = "truffleruby-gem-test-pack-#{TRUFFLERUBY_GEM_TEST_PACK_VERSION}"
+    unless Dir.exist?(test_pack)
+      puts "Downloading latest gem test pack..."
```

Comment:
This could interfere with the command's stdout. It could go to stderr to avoid the problem.